5.5/root/usr/share/container-scripts/mysql/validate-variables.sh

```diff
@@ -32,11 +30,34 @@ function validate_variables() {
     root_specified=1
   fi
 
+  # If MYSQL_USER == "root", we have a special case
+  if [[ "${user_specified:-0}" == "1" && "$MYSQL_USER" == "root" ]]; then
+    if [[ "${root_specified:-0}" == "1" ]]; then
+      usage "You cannot set MYSQL_USER to 'root' and use MYSQL_PASSWORD and MYSQL_ROOT_PASSWORD at the same time"
```

Comment:
shouldn't this just say "you can't set mysql_user to 'root' and use mysql_root_password at the same time?